kotlin-reference-server/src/main/kotlin/org/stellar/reference/di/ReferenceServerContainer.kt

```diff
@@ -0,0 +1,104 @@
+package org.stellar.reference.di
+
+import com.auth0.jwt.JWT
+import com.auth0.jwt.algorithms.Algorithm
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.auth.jwt.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import io.ktor.server.plugins.contentnegotiation.*
+import io.ktor.server.plugins.cors.routing.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.stellar.reference.callbacks.customer.customer
+import org.stellar.reference.callbacks.fee.fee
+import org.stellar.reference.callbacks.interactive.sep24Interactive
+import org.stellar.reference.callbacks.rate.rate
+import org.stellar.reference.callbacks.test.testCustomer
+import org.stellar.reference.callbacks.uniqueaddress.uniqueAddress
+import org.stellar.reference.data.AuthSettings
+import org.stellar.reference.event.event
+import org.stellar.reference.plugins.RequestExceptionHandlerPlugin
+import org.stellar.reference.plugins.RequestLoggerPlugin
+import org.stellar.reference.sep24.sep24
+import org.stellar.reference.sep24.testSep24
+
+const val AUTH_CONFIG_ENDPOINT = "endpoint-auth"
+
+object ReferenceServerContainer {
+  private val config = ConfigContainer.getInstance().config
```

Comment:
❓ Why call these a `container`?

Reply:
I think DI frameworks usually refer to a group of dependencies as a container.